docs/platform/limits/auth.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-displayed_sidebar: topLevelSidebar
+displayed_sidebar: platformSidebar
```

Comment:
FYI that for me the "collapse" arrow in the nav is getting clipped:

![image](https://github.com/user-attachments/assets/c31dcc44-28b5-4628-87a3-9c4493605523)